source/Mission.cpp

```diff
@@ -254,6 +255,20 @@ void Mission::Load(const DataNode &node)
 			else
 				child.PrintTrace("Skipping unrecognized attribute:");
 		}
+		else if(child.Token(0) == "subs")
+		{
+			for(const DataNode &grand : child)
+			{
+				if(grand.Size() > 1 && grand.Token(0).find(">") != string::npos)
+					child.PrintTrace("No \">\" allowed in subs keys (\""+grand.Token(0)+"\"):");
```

Comment:
If you print the trace from `grand` instead of `child`, you don't need to include any of its tokens in the trace string.

I would also prefer `"Skipping invalid use of angle brackets in substitution key:"` as the warning.